.github/workflows/ci.yml

```diff
@@ -0,0 +1,459 @@
+            php -r 'foreach (["silverstripe", "cwp", "symbiote", "dnadesign", "tractorcow", "bringyourownideas", "colymba"] as $v) { $d = "vendor/$v"; if (!is_dir($d)) continue; foreach (explode("\n", shell_exec("find $d | grep \/tests\/ | grep [a-zA-Z0-9]Test.php")) as $f) { if (preg_match("#ExtendTest\.php#", $f)) continue; if (is_file($f)) unlink($f); } }'
+            # Also remove a few other file that don't match the *Test.php convention and extends SapphireTest
+            if [ -f vendor/silverstripe/assets/tests/php/FilenameParsing/FileIDHelperTester.php ]; then
+              rm vendor/silverstripe/assets/tests/php/FilenameParsing/FileIDHelperTester.php
+            fi
+            if [ -f vendor/silverstripe/framework/tests/php/Forms/NullableFieldTests.php ]; then
+              rm vendor/silverstripe/framework/tests/php/Forms/NullableFieldTests.php
+            fi
+            if [ -f vendor/silverstripe/graphql/tests/Middleware/MiddlewareProcessTestBase.php ]; then
+              rm vendor/silverstripe/graphql/tests/Middleware/MiddlewareProcessTestBase.php
+            fi
+            # Rebuild composer classloader
+            composer dumpautoload -o
+          fi
+
+      - name: Final preparation
+        run: |
+          # Add .env file and create artifacts directory
+          if [[ "${{ matrix.db }}" =~ mysql ]]; then
+            if [ "${{ matrix.db }}" == "mysql57pdo" ]; then
+              cat << EOF > .env
+          SS_DATABASE_CLASS="MySQLPDODatabase"
+          SS_DATABASE_PORT="3357"
+          EOF
+            else
+              cat << EOF > .env
+          SS_DATABASE_CLASS="MySQLDatabase"
+          SS_DATABASE_PORT="${{ matrix.db == 'mysql57' && '3357' || '3380' }}"
+          EOF
+            fi
+            cat << EOF >> .env
+          SS_DATABASE_SERVER="127.0.0.1"
+          SS_DATABASE_USERNAME="root"
+          SS_DATABASE_PASSWORD="root"
+          EOF
+          else
+            cat << EOF > .env
+          SS_DATABASE_CLASS="PostgreSQLDatabase"
+          SS_DATABASE_SERVER="localhost"
+          SS_DATABASE_PORT="5432"
+          SS_DATABASE_USERNAME="postgres"
+          SS_DATABASE_PASSWORD="postgres"
+          EOF
+          fi
+          cat << EOF >> .env
+          SS_ENVIRONMENT_TYPE="dev"
+          SS_DATABASE_NAME="SS_mysite"
+          SS_DEFAULT_ADMIN_USERNAME="admin"
+          SS_DEFAULT_ADMIN_PASSWORD="password"
+          SS_TRUSTED_PROXY_IPS="*"
+          SS_MFA_SECRET_KEY="1234567894175b99966561e1efe237e4"
+          SS_BASE_URL="http://localhost"
+          EOF
+
+          # debug
+          echo ".env is"
+          cat .env
+
+          # Artifacts directory must be created after composer install as it would remove the artifacts directory
+          mkdir artifacts
+
+          # run dev/build flush to help debug any issues (though it's not strictly required here)
+          # travis-shared has no reference to sake, so it should be safe to omit dev/build flush=1
```

Comment:
This isn't travis. Also, I'm not sure what this comment is trying to say, even if it said "github actions has no reference...."? If it's just another comment to say "we don't _really_ need to do this" then remove the comment, it's already mentioned in the line above.

Reply:
Have removed comment, looks like it was a note to myself that no longer makes sense